### PR TITLE
schema for storing recommendation feedback

### PR DESCRIPTION
**Pull request by vansika**
This PR creates a schema to store user recommendation feedback.

### CODE REVIEW

File: admin/sql/create_types.sql [hunk 1]
```diff
@@ -3,3 +3,5 @@ CREATE TYPE cf_recording_type AS ENUM('top_artist', 'similar_artist');
 CREATE TYPE mb_missing_data_source_enum AS ENUM('cf', 'artist_map');
 
 CREATE TYPE user_relationship_enum AS ENUM('follow');
+
+CREATE TYPE feedback_type_enum AS ENUM('like_rec', 'love_rec', 'dislike_rec', 'hate_rec', 'bad_rec');
```
Comment (vansika) [inline admin/sql/create_types.sql:7]: @alastair these strings can be improved I think!

Comment (shivam-kapila) [reply admin/sql/create_types.sql:7]: Lets name it recommendation_feedback_type_enum as the other feedback doesnt use this

**Review by shivam-kapila — CHANGES_REQUESTED** [review]
Lgtm. A small change.

File: admin/sql/create_types.sql (hunk 1 above)
Comment (alastair) [reply admin/sql/create_types.sql:7]: I think that they would be fine without the `_rec` suffix. like, love, dislike, hate, bad_recommendation

**Review by alastair — APPROVED** [review]
Fine by me

File: admin/sql/create_tables.sql [hunk 2]
```diff
@@ -158,6 +158,14 @@ CREATE TABLE statistics.sitewide (
 
 ALTER TABLE statistics.sitewide ADD CONSTRAINT stats_range_uniq UNIQUE (stats_range);
 
+CREATE TABLE recommendation_feedback (
+    id                      SERIAL, -- PK
+    user_id                 INTEGER NOT NULL, -- FK to "user".id
+    recording_mbid          UUID NOT NULL,
+    rating                  feedback_type_enum NOT NULL,
```
Comment (alastair) [inline admin/sql/create_tables.sql:165]: wondering if we should have an index on this column ("get me all 'love' feedback for a user")

**Review by shivam-kapila — APPROVED** [review]
KGTM :)

**Comment by vansika** [comment]
Please don't merge this!

**Pull request merged**